app/code/Magento/Catalog/Test/Mftf/Test/AdminSortingByWebsitesTest.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminSortingByWebsitesTest">
+        <annotations>
+            <stories value="View sorting by websites"/>
+            <title value="Sorting by websites in Admin"/>
+            <description value="Sorting products by websites in Admin"/>
+        </annotations>
+        <before>
+            <actionGroup ref="LoginAsAdmin" stepKey="loginAsAdmin"/>
+            <!--Create new website -->
+            <actionGroup ref="AdminCreateWebsiteActionGroup" stepKey="createAdditionalWebsite">
+                <argument name="newWebsiteName" value="Second Website"/>
+                <argument name="websiteCode" value="second_website"/>
+            </actionGroup>
+        </before>
+        <after>
+            <actionGroup ref="AdminDeleteWebsiteActionGroup" stepKey="deleteTestWebsite">
+                <argument name="websiteName" value="Second Website"/>
```

Comment:
Once you will use `customWebsite` entity, you should also specify it here: `<argument name="websiteName" value="{{customWebsite.name}}"/>`
And also you should specify `{{customWebsite.name}}` in all actions you specified `'Second Website'`

Reply:
done